frontend/src/AudioAnnotator/AudioAnnotator.js

```diff
@@ -758,19 +755,59 @@ class AudioAnnotator extends Component<AudioAnnotatorProps, AudioAnnotatorState>
               {playbackRateSelect}
             </p>
 
-            <div className="col-sm-3 text-center">
-              <OverlayTrigger overlay={tooltip_submitAndSave}>
-                <button
-                  className="btn btn-submit"
-                  onClick={this.checkAndSubmitAnnotations}
-                  type="button"
-                >Submit &amp; load next recording</button>
-              </OverlayTrigger>
+            <div className="col-sm-5 text-center">
+            <OverlayTrigger overlay={
+              <div className="card">
+                <h3 className={`card-header tooltip-header`}>Shortcut</h3>
+                <div className="card-body p-1">
+                  <p>
+                    <span className="font-italic"><i className="fa fa-arrow-left"></i></span>{" : load previously recording"}<br/>
+                  </p>
+                </div>
+              </div>
+              }>
+              <Link className="btn btn-submit rounded-left rounded-right-0" disabled={this.state.task.prevAndNextAnnotation.prev === "" ? true : false} to={this.state.task.prevAndNextAnnotation.prev === "" ? "#" : `/audio-annotator/${ this.state.task.prevAndNextAnnotation.prev}`}>
+                <i className="fa fa-caret-left"></i>
+              </Link>
+            </OverlayTrigger>
+              <OverlayTrigger overlay={
+              <div className="card">
+                <h3 className={`card-header tooltip-header`}>Shortcut</h3>
+                <div className="card-body p-1">
+                  <p>
+                    <span className="font-italic">Enter</span>{" : Submit & load next recording"}<br/>
+                  </p>
+                </div>
+              </div>
+              }>
+              <button
+                className="btn btn-submit border-radius-0"
+                onClick={this.checkAndSubmitAnnotations}
+                type="button"
+              >Submit &amp; load next recording</button>
+            </OverlayTrigger>
+            <OverlayTrigger overlay={
+              <div className="card">
+                <h3 className={`card-header tooltip-header`}>Shortcut</h3>
+                <div className="card-body p-1">
+                  <p>
+                    <span className="font-italic"><i className="fa fa-arrow-right"></i></span>{" : load next recording"}<br/>
+                  </p>
+                </div>
+              </div>
+              }>
+              <Link className="btn btn-submit rounded-right rounded-left-0" disabled={this.state.task.prevAndNextAnnotation.next === "" ? true : false}
```

Comment:
Same as above:
We don't need a ternary operator here : `this.state.task.prevAndNextAnnotation.next === "" ? true : false`, the condition alone returns true or false.
And, just to improve readability, perhaps put each attribute on a new line?